server/src/server.ts

```diff
@@ -60,6 +61,29 @@ connection.onInitialize(
     },
 );
 
+connection.onHover((_params) => {
+    const doc: TextDocument = documents.get(_params.textDocument.uri);
+    const entityName = getTokenAtPosInDoc(doc.getText(), doc.offsetAt(_params.position));
+    if (!entityName) {
+        return null;
+    }
+    console.warn(`search for ${entityName}`);
+
+    const entity = completionProvider.getItemByLabel(entityName);
+    if (!entity) {
+        return null;
+    }
+    if (!entity.documentation) {
```

Comment:
LOL please use `||` to merge those two ifs.

Reply:
and then maybe an explicit `else` to host the `return { stuff }`.

Reply:
I merged them, but I won't add an explicit `else`. Both indentation and end of block brace are enough I think.